plutus/budget_manager/project_budget.py

```diff
@@ -0,0 +1,67 @@
+from constants import (
+    APP,
+    PLUTUS_CONFIG_TYPE_PROJECT,
+    PLUTUS_CONFIG_TYPE_PARENT,
+    PLUTUS_CONFIG_TYPE_LABEL,
+    PLUTUS_CONFIG_TYPE_DEFAULT
+)
+import logging
+import markus
+import sys
+
+metrics = markus.get_metrics(APP + ".plutus")
+
+class ProjectBudget:
```

Comment:
This looks like it could be a data class.

Reply:
Oh didnt know about these. The examples of dataclass i've seen thus far have the __init__ written for me), where as I wanted the passed in dictionary to be flattened into attributes of the class.

Reply:
I believe the default data class provides init but you can override it. It's a nice to have but not a blocker.